src/game.cpp

```diff
@@ -5362,7 +5364,7 @@ void game::draw_minimap()
                 mvwputch( w_minimap, 0, 3, c_red, '*' );
             }
         } else {
-            int arrowx = 3, arrowy = 3;
```

Comment:
Why are you making these un-initialized?

Reply:
Same as the reason for the space character above, "value is assigned but never used". Literally all the code does is write a 3 into that spot and then immediately overwrite it with something else.

Reply:
Fixed this, but github doesn't seem to realize the complaint is now outdated.